RealtimeJMSToStream/README.rst

```diff
@@ -0,0 +1,90 @@
+
+You can create and start the Adapter by using the CDAP CLI (or you can use the UI for a more visual approach).
+
+The realtime JMS source uses generic JMS interface to communicate with JMS broker, so the source requires implementation of the JMS interfaces
+to be able to work properly as separate adapter plugin.
+
+One important thing to remember you need to use the identical version of JMS for both the client and the target server to ensure 
+that the message format is respected when the JMS source start polling messages.
+
+The realtime JMS source requires an additional plugin jar that bundle all JMS specific implementations into a jar. 
+For Apache ActiveMQ, we will be using the ``activemq-all-5.11.1.jar`` file that contains most of the required dependencies.
+
+For each additionl plugin jars, there should be accompanying JSON file with the same name as the jar name, but with .json extension, 
+to describe about the plugin so the adapter knows about it and load it as part of the plugon class loader. The jar and the JSON counterpart files
+should be put in the templates/plugins/ETLRealtime directory.
+
+There are additional jars that should be copied into templates/plugins/ETLRealtime/lib directory. These jars will be available as part of combined 
+class loaders for all realtime plugins so need to copy jar containing JMS specifications API, for example 
+
+Available Properties for JMS Source
+===================================
+
+To configure the source, there are some properties that are needed to make it work properly:
+
+#. ``jms.destination.name``: Name of the JMS destination (either Topic or Queue name) to get messages
+
+#. ``jms.messages.receive``: This is the Max number messages should be retrieved per poll. The default value is 50
+
+#. ``jms.factory.initial``: The fully qualified class name of the factory class that will create an initial context
+
+#. ``jms.provider.url``: This is the URL of the JMS server broker to connect to
+
+#. ``ConnectionFactory``: Change the format of the name for the connectionfactory in JMS source plugin. The default value is ConnectionFactory. 
+
+In addition to the properties expected by the JMS source, you could always pass adiitional key-value pairs of properties that will be 
+passed to underlyong JNDI initial context to configure the implementation of the JMS APIs.
+For example, the Apache ActiveMQ requires the Topic or Queue name to be set as topic.<destination name> or queue.<destination name> property key names
+in order for the JNDI context to be resolved.
+
+The ``config.json`` file included shows the Adapter configuration file with sample values for the JMS source properties.
+
+The JMS APIs Implementation Plugin JSON Configuration File
+===========================================================
+
+As mentioned before, the JMS source requires additional plugin to actually does the execution of polling the messages from the target server.
```

Comment:
As mentioned above, the JMS source requires an additional plugin that actually performs the execution of polling messages from the target server.